typemap

```diff
@@ -0,0 +1,2 @@
+double *					T_PTROBJ
```

Comment:
IMHO you should not export the double* at all, but use native perl type for floats `NV`